kani-driver/src/main.rs

```diff
@@ -45,11 +46,12 @@ fn cargokani_main(input_args: Vec<OsString>) -> Result<()> {
     if matches!(args.command, Some(CargoKaniSubcommand::Assess)) || ctx.args.assess {
         // --assess requires --enable-unstable, but the subcommand needs manual checking
         if !ctx.args.enable_unstable {
-            clap::Error::raw(
-                clap::error::ErrorKind::MissingRequiredArgument,
-                "Assess is unstable and requires 'cargo kani --enable-unstable assess'".to_string(),
-            )
-            .exit()
+            args::CargoKaniArgs::command()
```

Comment:
I'm assuming this won't print the command context, right?

Reply:
This is what it prints:
```
error: Assess is unstable and requires 'cargo kani --enable-unstable assess'

Usage: cargo-kani [OPTIONS]

For more information try '--help'
```